minecraft_servers/thelotus/manifest.json

```diff
@@ -0,0 +1,28 @@
+{
+    "server_name": "thelotus",
+    "nice_name": "TheLotus",
+    "direct_ip": "thelotus.tv",
+    "server_wildcards": [
+        "mc.thelotus.tv"
+    ],
+    "supported_languages": [
+        "en",
+        "de"
+    ],
+    "social": {
+        "web": "https://thelotus.tv/",
+        "web_support": "https://thelotus.tv/forum/",
```

Comment:
Please link a direct support page or the support category in your forum. The forum itself is not a support platform.